diff --git a/node_modules/validate-html-nesting/src/mapping.js b/node_modules/validate-html-nesting/src/mapping.js
index 323b7d6..d9edfef 100644
--- a/node_modules/validate-html-nesting/src/mapping.js
+++ b/node_modules/validate-html-nesting/src/mapping.js
@@ -18,7 +18,7 @@ const onlyValidChildren = {
 		'template',
 	]),
 	optgroup: new Set(['option']),
-	select: new Set(['optgroup', 'option']),
+	select: new Set(['optgroup', 'datalist', 'button', 'option']),
 	math: new Set(['mrow']),
 	script: new Set(),
 	// table
